src/components/KeyboardView.js

```diff
@@ -1,6 +1,8 @@
 // @flow
-import React, { PureComponent } from "react";
+import React from "react";
 import { KeyboardAvoidingView, Platform, NativeModules } from "react-native";
+import * as ExperimentalHeader from "../screens/Settings/Experimental/ExperimentalHeader";
```

Comment:
maybe import directly  HEIGHT if you're only using it here :thinking: 
`import { HEIGHT as ExperimentalHeaderHeight } from "../screens/Settings/Experimental/ExperimentalHeader";`